Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,75 @@
 import React from 'react';
 import './App.css';
+//component imports
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import OperationButton from './components/ButtonComponents/OperationButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
 
 const App = () => {
+//number declarations
+const one = 1;
+const two = 2;
+const three = 3;
+const four = 4;
+const five = 5;
+const six = 6;
+const seven = 7;
+const eight = 8;
+const nine = 9;
+const clear = 'clear';
+const zero = 0;
+//operation declarations
+const divide = '÷';
+const mult = 'X';
+const subtract = '-';
+const add = '+';
+const equals = '=';
+
+//button class names for props
+const buttonStyle = 'btn';
+const action = 'action-btn';
+const action2 = 'clear-text';
+const opBtn = 'op-btn';
+//display class name
+const displayClass = 'calc-display';
+
+
   return (
     <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
+
+    <div className = 'calc-container'>
+      <CalculatorDisplay displayClass = {displayClass} text = {zero} />
+      
+        <div className ='row-1'>
+          <ActionButton  action = {action2} text = {clear} />
+          <OperationButton opBtn = {opBtn} text = {divide} />
+        </div>
+          <div className = 'row-2'>
+            <NumberButton buttonStyle = {buttonStyle} text = {seven} />
```

Comment:
Instead of referencing a const you defined above, you can hard-code the numbers and symbols into the button text (to save time and lines of code). DRY code!